crate/kmip/src/openssl/public_key.rs

```diff
@@ -89,6 +89,7 @@ pub fn kmip_public_key_to_openssl(public_key: &Object) -> Result<PKey<Public>, K
                 recommended_curve,
                 q_string,
             } => match recommended_curve {
+                // P-CURVES
                 RecommendedCurve::P192 => {
```

Comment:
```suggestion
                #[cfg(not(feature = "fips"))]
                RecommendedCurve::P192 => {
```

Reply:
done in 4d5a656a2044162d9356e159bc8723c274db3497